decompiler/config/jak1_ntsc_black_label.jsonc

```diff
@@ -4,6 +4,7 @@
   // if you want to filter to only some object names.
   // it will make the decompiler much faster.
   "allowed_objects": [],
+  "banned_objects": ["crates"],
```

Comment:
whats the point of banning crates? can't we just not add it to our tests until its ready to go

Reply:
It's for when I want to run the decompiler on all files for xreferencing symbol usage.